oauth2.md

```diff
@@ -114,19 +114,26 @@ POST https://oauth.mocaverse.xyz/api/oauth/token
 ```
 Content-type: `application/x-www-form-urlencoded`
 
-| Parameter       | Description                                                                            |
-|-----------------|----------------------------------------------------------------------------------------|
-| grant_type    | must be `authorization_code`                                                           |
-| client_id     | your `client_id` as provided during [Set up](#set-up)             |
-| client_secret | your `client_secret` as provided during [Set up](#set-up)          |
+| Parameter     | Description                                                                           |
+|---------------|---------------------------------------------------------------------------------------|
+| grant_type    | must be `authorization_code`                                                          |
+| client_id     | your `client_id` as provided during [Set up](#set-up)                                 |
+| client_secret | your `client_secret` as provided during [Set up](#set-up)                             |
 | code          | the authorization code received from [step 2](#2-authorization-code-request-callback) |
-| redirect_uri  | must match the value used during [step 1](#1-user-authorization-request)          |
+| redirect_uri  | must match the value used during [step 1](#1-user-authorization-request)              |
 
 ##### Example
 
 ```http
-POST https://oauth.mocaverse.xyz/api/oauth/token?grant_type=authorization_code&client_id=EXAMPLE_CLIENT_ID&client_secret=MOCA_SECRET&code=602cbbe1-60dd-482f-b0d8-329f5f1254c7&redirect_uri=https%3A%2F%2Fyour-app.example.com%2Fcallback
+POST https://oauth.mocaverse.xyz/api/oauth/token
+```
+
+**Request Body**
+
+```
+grant_type=authorization_code&client_id=EXAMPLE_CLIENT_ID&client_secret=MOCA_SECRET&code=602cbbe1-60dd-482f-b0d8-329f5f1254c7&redirect_uri=https%3A%2F%2Fyour-app.example.com%2Fcallback
 ```
+
```

Comment:
Relevant change. Others are automatically formatted tables.

Reply:
also add header in this example

Reply:
Updated